src/Room.src.json

```diff
@@ -0,0 +1,43 @@
+{
+    "type": "Room",
+    "uri": "http://schema4i.org/Room",
+    "description": "A room is a distinguishable space within a structure, usually separated from other spaces by interior walls.",
+    "links": [{
+        "url": "http://schema.org/Room",
+        "description": "Original Schema.org type"
+    }],
+    "parents": [
+        
```

Comment:
Is it right that there's no parent using this object? A possible parent could be `Building.ContainsPlace`. Please check and place another pull request for this.